glide-core/src/client/value_conversion.rs

```diff
@@ -175,6 +176,64 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        ExpectedReturnType::Lolwut => {
+            match value {
+                // cluster (multi-node) response
+                Value::Map(map) => {
```

Comment:
Do you know how ping handles this call?  Maybe we should do something similar, since the return should be the same for the entire cluster.

Reply:
Good question. All commands other commands that require data conversion have no cluster implementation.
Testing...

Reply:
this crashes in glide core
```java
clusterClient.customCommand(new String[] { "SMEMBERS", "s1" }, ALL_NODES).get()
```
there is a conversion for `SMEMBERS`, but it does not handle map case

Reply:
`ping` runs against all nodes, but aggregates the result into a single string.

Reply:
it happens in `redis-rs`